poller/README.md

```diff
@@ -84,6 +84,7 @@ Key                      | Default Value  | Description
 `scaleInCoolingMinutes`  | 30             | Minutes to wait after scaling IN or OUT before a scale IN event can be processed.
 `overloadCoolingMinutes` | 5              | Minutes to wait after scaling IN or OUT before a scale OUT event can be processed, when the Spanner instance is overloaded. An instance is overloaded if its High Priority CPU utilization is over 90%.
 `stateProjectId`         | `${projectId}` | The project ID where the Autoscaler state will be persisted. By default it is persisted using [Cloud Firestore][cloud-firestore] in the same project as the Spanner instance.
+`stateDatabase`          | Object         | An Object that can override the database for managing the state of autoscale. The default database is Firestore. Refer to the [state database](#state-database) to see for the detail.
```

Comment:
the state of the Autoscaler.

Refer to [state database](#state-database) for details.